src/main/resources/reference.conf

```diff
@@ -72,13 +74,13 @@ kamon {
     information-unit = "b"
 
     additional-tags {
-      service = "yes"
-      host = "yes"
-      instance = "yes"
-      blacklisted-tags = []
+      include-service = "yes"
+      include-host = "yes"
+      include-instance = "yes"
+      exclude = []
     }
 
-    filter-config-key = "datadog-tag-filter"
+    filter-config-key = "kamon.util.filters.datadog-tag-filter"
```

Comment:
We are no longer using `kamon.util.filters` on RC2. What about removing this key and renaming `additional-tags` to `environment-tags`? just one setting should be enough for that.

Reply:
I'm not sure I understand what you're looking for exactly. Are you saying don't include the filter at all? I'm actually not a fan of what I currently have. It means you can't have different settings for the different reporters.